### DISCUSSION TITLE
What about refresh token?

### DISCUSSION THREAD

**Discussion by Bibazavr** [payloadcms/payload#2601] [Feature Requests & Ideas]
Looks like payload cms works only with 1 token 

What do think about add refresh token in auth for [better user experience](https://www.loginradius.com/blog/identity/refresh-tokens-jwt-interaction/#:~:text=The%20API%20returns%20a%20short,it%20expires%20or%20even%20before.)

**Comment by matthijs166** [comment]
What is the benefit of adding this into payload?
